internal/pkg/print/pretty/pretty.go

```diff
@@ -52,7 +52,7 @@ const (
 			{{- printf "\n" -}}
 			{{- range . }}
 				{{ printf "output.%s" .Name | colorize "\033[36m" }}
-				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}
+				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}{{ if $.Settings.OutputValues }}{{- printf "\n" -}}{{ .Value | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}{{ end }}
```

Comment:
To keep it consistent with input values, I think this would work better. this will show the output value in `(...)` in front of the output name and without any color, the same as inputs.

```suggestion
				{{- if $.Settings.OutputValues -}}
					{{- printf " " -}}
					({{ .Value | trimSuffix "\n" | default "n/a" }})
				{{- end }}
				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}
```

Reply:
- [X] Make line 55 prettier